services/observation_plan_queue/observation_plan_queue.py

```diff
@@ -22,6 +24,26 @@
 
 Session = scoped_session(sessionmaker())
 
+REQUEST_TIMEOUT_SECONDS = cfg['health_monitor.request_timeout_seconds']
```

Comment:
Maybe this could be a get with a default?

Reply:
I would not actually. This is a basic baselayer config argument we should also have in multiple places of the app, and we must be using the right value. You can imagine a user not passing the config right and we want it to fail if that happens I think.